lib/web/css/source/lib/_utilities.less

```diff
@@ -14,7 +14,7 @@
 .lib-font-size-value(
     @_value
 ) when not (@_value = false) and not (@_value = '') and (@font-size-unit-convert) {
-    @fontValue: unit(((@_value) * 1), @font-size-unit) / @font-size-unit-ratio;
+    @fontValue: (unit(((@_value) * 1), @font-size-unit) / @font-size-unit-ratio);
```

Comment:
Are you sure it sohuldn't be inside the unit function call?

Reply:
https://lesscss.org/usage/#less-options-math
imho in this case we need parentheses. Specially when use less4